### ISSUE TITLE
Cross-env-shell doesn't work

### ISSUE THREAD

**Issue by JimTheMan** [kentcdodds/cross-env#184]
- `cross-env` version: 5.2.0
- `node` version: 8.11.1
- `npm` (or `yarn`) version: 5.6.0

Relevant code or config

```javascript

```

What you did:

In package.json:

  "scripts": {
    "set-env": "cross-env-shell NODE_ENV=development",
}

Then in my shell I run `node` and enter `process.env.NODE_ENV`.

What happened:

NODE_ENV has no value.

Reproduction repository:

Problem description:

Suggested solution:

NODE_ENV should have a value...

**Comment by singhsoldier13** [comment]
+1

**Comment by JimTheMan** [comment]
@singhsoldier13 I don't think this library is meant to have the env variables still available after the current node session (and basically each node command you run is treated as a different session).

Too bad by @kentcdodds is not paying attention to this project... can someone please confirm this?
